test/gtest/uct/ib/test_ud.cc

```diff
@@ -368,7 +368,7 @@ UCS_TEST_SKIP_COND_P(test_ud, ack_req_window,
 
 /* simulate retransmission of the CREQ packet */
 UCS_TEST_SKIP_COND_P(test_ud, crep_drop1,
-                     !check_caps(UCT_IFACE_FLAG_PUT_SHORT)) {
+                     !check_caps(UCT_IFACE_FLAG_AM_SHORT)) {
```

Comment:
why do we have to require any flag? seems this test does not even call uct_ep_am_short

Reply:
to avoid failures on OPA

it sends short (inlined messages) with UD

Reply:
ok, AM_SHORT as indirect indication of put_short..